src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static String line = "__________________________________________________________";
+    private static Task[] inputList = new Task[101];
+    private static int numTasks = 0;
+
+    public static void printAddTask(Task t) {
```

Comment:
```suggestion
    public static void printAddedTask(Task task) {
```
print add might be might make the reader think that both printing and adding will be done in the function whereas only printing will be done in this function.